src/common/options.html

```diff
@@ -282,6 +282,18 @@ <h1 class="page-header">
 
                           <div class="container col-xs-11">
                             <div class="row col-xs-10">
+                              <div class="col-xs-2">
+                                <input type="checkbox" id="daysOfBuffering" name="daysOfBuffering" aria-describedby="daysOfBufferingHelpBlock">
+                              </div>
+                              <div class="col-xs-8">
+                                <label for="daysOfBuffering">Add Days of Buffering Metric</label>
+                                <span id="daysOfBufferingHelpBlock" class="help-block">In the default move money dialog you have to use the mouse to select where you want it to go to. This option adds the ability to type to find the category.</span>
```

Comment:
Can you change the description here? This looks to be copy+pasted from another entry.